ruby_event_store/lib/ruby_event_store/subscription.rb

```diff
@@ -15,6 +17,10 @@ def unsubscribe
       event_types.each{ |type| @store.delete(self, type) } if @store
     end
 
+    def global?
+      event_types.include?(GLOBAL_SUBSCRIPTION)
```

Comment:
Is it always `event_types == [GLOBAL_SUBSCRIPTION]` or do we allow a `Subscription` to be both global and for some particular type?